Tests/IGListAdapterTests.m

```diff
@@ -751,13 +751,36 @@ - (void)test_whenScrollVerticallyToBottom_withContentInsets_thatBottomFlushWithC
     self.collectionView.contentInset = UIEdgeInsetsMake(0, 0, 100, 0);
     [self.collectionView layoutIfNeeded];
     [self.adapter scrollToObject:@100 supplementaryKinds:nil scrollDirection:UICollectionViewScrollDirectionVertical scrollPosition:UICollectionViewScrollPositionBottom animated:NO];
-    IGAssertEqualPoint([self.collectionView contentOffset], 0, 900);
+    IGAssertEqualPoint([self.collectionView contentOffset], 0, 900 + 100);
 
     // top 50, bottom 100
     self.collectionView.contentInset = UIEdgeInsetsMake(50, 0, 100, 0);
     [self.collectionView layoutIfNeeded];
     [self.adapter scrollToObject:@100 supplementaryKinds:nil scrollDirection:UICollectionViewScrollDirectionVertical scrollPosition:UICollectionViewScrollPositionBottom animated:NO];
-    IGAssertEqualPoint([self.collectionView contentOffset], 0, 900);
+    IGAssertEqualPoint([self.collectionView contentOffset], 0, 900 + 100);
+    
+    UICollectionViewLayout *layout = self.collectionView.collectionViewLayout;
+    if ([layout isKindOfClass:[UICollectionViewFlowLayout class]]) {
```

Comment:
Can you change this to a hard cast so the test crashes if it changes?

Reply:
fixed in 7ed9b9757140f235b5dc9751d94b394b12536033